### PR TITLE
Merge working branch to master

### PR DESCRIPTION
**Pull request by fc-anjos**
This pull request merges all development, performed on aside branch, to master.
The development was implemented in the following phases, as described in the commit history:

- Adding linters config
- General Structure of HTML
- Building blocks of CSS
- Advancing on HTML and CSS
- Adding media queries
- Minor cleanup
- Adding README.md

### CODE REVIEW

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team   
#### Great Job so far, but there are few more things required. 

### Page structure
#### Navigation bar

- [ ]  The NY Times logo should be a link, please fix that.
- [ ] You are to use [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section .
- [ ] There are no images displayed in the **Theory of Inflation** section, please let's fix that too.
##### Original Version


##### Your Version


#### General
- [ ] Concerning the comments in your code: It’s better to delete commented code. If you need to review a previous version of your code, use git history

### Make changes and submit another code review request.
*Happy coding*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1147 @@
...
+              page and drew a double box around it.
+            </p>
+            <p class="paragraph">
+              On Monday, Dr. Guth’s starship came in. Radio astronomers reported
+              that they had seen the beginning of the Big Bang, and that his
+              hypothesis, known undramatically as inflation, looked right.
+            </p>
+          </div>
+
+          <div class="aside_container">
```
Comment (euqueme) [inline index.html]: change the **\<div\>** tag for an **\<aside\>**  tag in line #219 

> \<aside class="aside_container"\>

**Review by euqueme — CHANGES_REQUESTED** [review]
>### Pull Request under the review by Maria Eugenia Quemé P. (TSE)

### Hello @fc-anjos and @phfdonda  :handshake: 

### This project has great attention to detail :microscope: thank you for doing the previous changes :medal_sports: there are still some mandatory changes required to be able to give it the approval :memo: 

**Bottom part**

- [ ]  the articles at the More in space and technology and Editor's picks should be links to other articles  (you can surround them with \<a\> tags to fix them)

#### Project specific

#### Implementation details

* [ ] It uses [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

[Optional] your Live Demo doesn't seem to be working please address these [steps](https://help.github.com/en/enterprise/2.13/user/articles/configuring-a-publishing-source-for-github-pages)

I have commented your code with a suggestion where the Aside tag should be in the comments bellow :arrow_down: 

### That's it, team! :raised_hands: 
### please :pray: make the changes and submit back for a code review :rocket: 

Happy coding! :woman_technologist: 

STATUS OF THE PROJECT: REQUESTED CHANGES :vertical_traffic_light:

**Comment by fc-anjos** [comment]
Thanks Maru and Adama for the reviews,

We are now committing a version that incorporates all the required changes.
As for the live-preview, we have set up the link so that it will work when the readme is merged - along all necessary files - to master branch.

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**